src/index.html

```diff
@@ -13,6 +13,76 @@
     />
   </head>
   <body>
```

Comment:
Remember about correct indentation between parent and child elements. The `<div>` tag should be indented with 2 spaces relative to its parent `<body>` tag.